api/src/db/migrations/v152_add_land_to_environment_data.test.ts

```diff
@@ -80,7 +80,6 @@ describe("v152_add_land_to_environment_data", () => {
   it("adds land section to environment data", () => {
     const id = "biz-1";
     const v151Business = generatev151Business({
-      taskProgress: { "waste-permitting": "IN_PROGRESS" },
```

Comment:
Why did we remove this?

Reply:
@jphechter I took it out because it's not necessary for the test that the waste task is in progress, it's not relevant to what we are trying to test. We just want to ensure that the land section gets added.